### ISSUE TITLE
error while runnig "/react/components/rating" from docs in localhost environment

### ISSUE THREAD

**Issue by saketverma-03** [aws-amplify/amplify-ui#4299]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have tried disabling all browser extensions or using a different browser
- [X] I have tried deleting the node_modules folder and reinstalling my dependencies
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Other

### How is your app built?

react

### What browsers are you seeing the problem on?

Chrome, Firefox

### Which region are you seeing the problem in?

/react/components/rating  page in docs

### Please describe your bug.

![Screenshot from 2023-07-29 19-34-55](https://github.com/aws-amplify/amplify-ui/assets/63857530/2940e90e-d7a2-4c28-8837-dc4697ec8689)

i start the docs in localhost using `yarn docs dev`

I tried to track down the issue and this is what i found
[this](https://github.com/aws-amplify/amplify-ui/blob/448b7859387a1ff22244ee464d68bd6b448d0ef5/docs/src/pages/%5Bplatform%5D/components/rating/index.page.mdx#L15) line seems to be causing the issue.

Its trying to  `import data from '@/data/props-table.json';` which does not exist in `docs/src/data` folder

### What's the expected behaviour?

[this](https://ui.docs.amplify.aws/react/components/rating) page should load

### Help us reproduce the bug!

To reproduce , simply follow the contribution guideline for docs,then run it in local machine and try navigate to `react/components/rating`

### Code Snippet

```typescript
// Put your code below this line.

```

**Comment by hbuchel** [comment]
@saketverma-03 If you need to unblock in the meantime, you can run `yarn docs build` and it will run the necessary commands to build the props and css table json files and the docs site. Alternatively, you can run `yarn build:cssvars-table && yarn build:props-tables` to just build just the table json files before running `yarn docs dev`. You should just need to build those tables once if you are just looking at docs updates.
